auth/auth_test.go

```diff
@@ -87,8 +89,8 @@ func TestNewClientWithServiceAccountCredentials(t *testing.T) {
 	if _, ok := client.signer.(*serviceAccountSigner); !ok {
 		t.Errorf("NewClient().signer = %#v; want = serviceAccountSigner", client.signer)
 	}
-	if client.projectID != creds.ProjectID {
-		t.Errorf("NewClient().projectID = %q; want = %q", client.projectID, creds.ProjectID)
+	if err := checkIDTokenVerifier(client.idTokenVerifier, creds.ProjectID); err != nil {
+		t.Errorf("NewClient().idTokenVerifier = %v; want = nil", err)
```

Comment:
This message seems a little odd. Are you sure this is what you meant?

Reply:
Changed to `t.Errorf("NewClient().idTokenVerifier: %v", err)`